library/src/main/java/com/novoda/downloadmanager/DownloadBatch.java

```diff
@@ -46,7 +46,7 @@ void setCallback(DownloadBatchStatusCallback callback) {
 
     void download() {
         DownloadBatchStatus.Status status = downloadBatchStatus.status();
-        if (status == PAUSED || status == DELETION || status == DOWNLOADED) {
```

Comment:
This was added to get around a double call of notifications, when a file is marked as downloading and then again when the download is immediately marked as downloaded. Instead we should handle this as a new flag in the db, we still have outstanding issues with notifications anyway that this didn't fix that the db solution would.